src/test/java/util/SupportUtils.java

```diff
@@ -38,6 +45,11 @@
 * byte[] csvBytes = getBytesFromFile(pathToCsv);
 * processCsvAndDump(csvBytes);
 * }
+* // Read a CSV file, parse it, turning each CSV record into
```

Comment:
Would you be able to change the comments to make them more readable please as there are comment tags within the javadoc?